fonts/icons/material-symbols-outlined/metadata.json

```diff
@@ -0,0 +1 @@
+{"fontId":"material-symbols-outlined","fontName":"Material Symbols Outlined","subsets":[],"weights":[],"styles":[],"defSubset":"","variable":true,"lastModified":"2022-08-11","version":"v1","category":"display","source":"https://fonts.google.com/icons","license":"https://www.apache.org/licenses/LICENSE-2.0","type":"icons"}
```

Comment:
I manually edited the subsets, weights, styles, and defSubset props after running the generator